### PR TITLE
style(app:) Capitalize 96 ch exit text

### PR DESCRIPTION
**Pull request by mjhuff**
# Overview

Capitalize 96 ch exit text during the attach flows.

image

# Test Plan
- Verify text is capitalized.

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ describe('UnskippableModal', () => {
     screen.getByText(
       'You must detach the mounting plate and reattach the z-axis carraige before using other pipettes. We do not recommend exiting this process before completion.'
     )
+    screen.getByText('Exit')
+    screen.getByText('Go back')
     fireEvent.click(screen.getByRole('button', { name: 'exit' }))
```
Comment (koji) [inline app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx]:
```ts
fireEvent.click(screen.getByRole('button', { name: 'Exit' }))
```

File: app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx [hunk 2]
```diff
@@ -39,6 +39,8 @@ describe('UnskippableModal', () => {
     screen.getByText(
       'You must detach the mounting plate and reattach the z-axis carraige before using other pipettes. We do not recommend exiting this process before completion.'
     )
+    screen.getByText('Exit')
```
Comment (koji) [inline app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx:42]: The change totally makes sense to me and I'm not sure why this PR gets error. 🤔 

```shell
            <p
                  class="sc-eDWCr sc-hTBuwn fkMzKx"
                >
                  exit
                </p>
              </div>
            </button>
            <button
              class="sc-gswNZR sc-jfvxQR bfAhEf cgckGb"
              type="button"
            >
              <div
                class="sc-jrcTuL eVSelp"
              >
                <p
                  class="sc-eDWCr sc-hTBuwn fkMzKx"
                >
                  Go back
                </p>
              </div>
            </button>
          </div>
        </div>
      </div>
    </body>

      40 |       'You must detach the mounting plate and reattach the z-axis carraige before using other pipettes. We do not recommend exiting this process before completion.'
      41 |     )
    > 42 |     screen.getByText('Exit')
         |            ^
      43 |     screen.getByText('Go back')
      44 |     fireEvent.click(screen.getByRole('button', { name: 'exit' }))
      45 |     expect(props.proceed).toHaveBeenCalled()
```

File: app/src/organisms/PipetteWizardFlows/UnskippableModal.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export function UnskippableModal(props: UnskippableModalProps): JSX.Element {
           <SmallButton
             marginRight={SPACING.spacing8}
             onClick={proceed}
-            buttonText={t('shared:exit')}
+            buttonText={i18n.format(t('shared:exit', 'capitalize'))}
```
Comment (koji) [inline app/src/organisms/PipetteWizardFlows/UnskippableModal.tsx]: missed `)` 
```suggestion
            buttonText={i18n.format(t('shared:exit'), 'capitalize')}
```

File: app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx (hunk 2 above)
Comment (koji) [reply app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx:42]: never mind. figured out what the issue is. I left a comment ↑

**Comment by mjhuff** [comment]
This is what I get for pushing a PR while walking out the door 😄

File: app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx (hunk 2 above)
Comment (mjhuff) [reply app/src/organisms/PipetteWizardFlows/__tests__/UnskippableModal.test.tsx:42]: Very weird that the debugger fails at L42 🧐  Thanks for catching that.

**Review by koji — APPROVED** [review]
LGTM!

**Pull request merged**